ui/src/dialogs/AuthenticationDialog/AuthenticationDialog.scss

```diff
@@ -22,8 +18,8 @@
   }
 
   .link-box {
-    margin-top: 2em;
-    margin-bottom: 2em;
-    // margin-left: 8.5em;
+    // TODO: This is a hacky way to center the link horizontally. We should
+    // refactor this and use flexbox to achieve that.
```

Comment:
Any chance you could create an issue for that and reference it in this comment?

Reply:
To be honest, there are many cases similar to this one in our code. I’m not sure we will refactor them anytime soon (or at all), and the value of refactoring it is very limited as long as we do not plan to change this specific UI significantly.

If I open an issue for every case like this, we’d end up with a lot of issues that will see no activity for a very long time and add additional clutter to our issues list, so I’d prefer not to do that, but not sure if I’m missing something and am happy to discuss!